__tests__/app.test.js

```diff
@@ -51,3 +51,44 @@ describe('/api', () => {
             });
     });
 });
+
+describe('/api/articles/:article_id', () => {
+    test('GET - status: 200 - responds with article with specified id', () => {
+        const testArticle = {
```

Comment:
This is fine if we know the data structure and that it will stay consistent, however this test would break later if another key was added later to the article. Using matchers like we have during the course would be better idea, asserting against keys, using toMatchObject etc

Reply:
Thanks for the feedback. Do I need to merge this request before I can submit another request? I made the changes and pushed them to the same branch, but I don't see a new merge request coming up.